dependencies.yaml

```diff
@@ -0,0 +1,75 @@
+options:
+  buildHeader: 
+    - "load(\"@io_bazel_rules_scala//scala:scala.bzl\", \"scala_library\")\nload(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")"
+  languages: [ "java", "scala:2.12.7" ]
+  resolverType: "coursier"
+  resolvers:
+    - id: "mavencentral"
+      type: "default"
+      url: https://repo.maven.apache.org/maven2/
+  strictVisibility: true
+  transitivity: exports # runtime_deps
+  versionConflictPolicy: highest
+
+dependencies:
+
+
+
+  com.github.pureconfig:
+    pureconfig:
```

Comment:
My understanding is that pureconfig, nor typesafeconfig are required.

Reply:
Now I see that pureconfig is actually used to auto-derive codecs for the crypto classes, I still believe typesafe config is not required.

Reply:
I will check this